skills/zap/index.js

```diff
@@ -0,0 +1,55 @@
+const http = require('http');
```

Comment:
Lots of tab indents in this file, replace them with spaces. Also our overlord Pat has something against semi-colons :)